playbook/app/pb_kits/playbook/pb_text_input/_text_input.jsx

```diff
@@ -89,6 +89,7 @@ const TextInput = (props: TextInputProps, ref: React.ElementRef<"input">) => {
         className="text_input"
         disabled={disabled}
         id={id}
+        key={`${id}_${className}`}
```

Comment:
Will this work with assigning `id` only? Just curious what className will do here?

Reply:
it was acting weird when i used only the id, so i added the classname just to be safe. Just to ensure uniqueness pretty much